packages/facebook-config-ui/package.js

```diff
@@ -0,0 +1,14 @@
+Package.describe({
+  summary: "Blaze configuration templates for Facebook OAuth.",
+  version: "1.0.0"
+});
+
+Package.onUse(function(api) {
+  api.use('templating@1.2.13', 'client');
+  api.use('accounts-facebook@1.1.0');
```

Comment:
I'm not sure this package should depend on accounts facebook.

a) it leads to a circular dependency
b) the `facebook` package didn't previously
c) in theory you could have the config ui w/o accounts ?